src/Stepper/Step.jsx

```diff
@@ -0,0 +1,336 @@
+import React, {PropTypes} from 'react';
+
+import {
+  TouchRipple,
+} from '../ripples';
+
+import Avatar from '../avatar';
+import Mixins from '../mixins';
+
+import {getMuiTheme} from '../styles';
+
+const Step = React.createClass({
+  propTypes: {
+    children: PropTypes.node,
+
+    /**
+     *Override the inline-styles of div which contains all the children include control button groups
+     */
+    childrenWrapperStyle: PropTypes.object,
+    /**
+     *Override the inline-styles of connector line
+     */
+    connectorLineStyle: PropTypes.object,
+    /**
+     *An array of node for handling moving or canceling steps
+     */
+    controlButtonsGroup: PropTypes.arrayOf(PropTypes.node),
```

Comment:
I agree with you. But when i implemented controlButtons like array of Node, because i wanted to set default style for the wrapper of button control group (like material ui design). They can pass by area of buttons, or one by one.